pkg/cmd/api/api_test.go

```diff
@@ -912,7 +912,7 @@ func Test_apiRun_cache(t *testing.T) {
 	err = apiRun(&options)
 	assert.NoError(t, err)
 
-	assert.Equal(t, 1, requestCount)
+	assert.Equal(t, 2, requestCount)
```

Comment:
Not sure that this test is valid anymore since the caching happens on the `go-gh` side.